src/test/scala/uk/gov/hmrc/test/ui/pages/section6/RemoveContainerPage.scala

```diff
@@ -20,7 +20,7 @@ import uk.gov.hmrc.test.ui.pages.base.BasePage
 import uk.gov.hmrc.test.ui.pages.base.Constants.yesNo
 import uk.gov.hmrc.test.ui.pages.section6.DetailKeys.Container
 
-object ContainerRemovePage extends BasePage {
+object RemoveContainerPage extends BasePage {
```

Comment:
For consistency this should actually be `ContainerRemovePage`

Reply:
Done